components/LandingPage.tsx

```diff
@@ -98,7 +98,7 @@ const LandingPage: React.FC = () => {
               alt="Work Experience"
               layout="fixed"
             />
-            <h4 className="mt-3 fw-bold">Work Experience</h4>
+            <p className="mt-3 fw-bold h3">Work Experience</p>
```

Comment:
Does using `p` instead of a header for this file cause any change in UI?

Reply:
No, since it gets the `h3` header css from the `h3` classname.